skaffold-operator.yaml

```diff
@@ -1,253 +1,245 @@
+            - pkg/resourcemanager/webhook/seccompprofile
+            - pkg/resourcemanager/webhook/systemcomponentsconfig
+            - pkg/resourcemanager/webhook/tokeninvalidator
+            - pkg/utils
+            - pkg/utils/context
+            - pkg/utils/errors
+            - pkg/utils/flow
+            - pkg/utils/gardener
+            - pkg/utils/kubernetes
+            - pkg/utils/kubernetes/health
+            - pkg/utils/retry
+            - pkg/utils/secrets
+            - pkg/utils/timewindow
+            - pkg/utils/validation/admissionplugins
+            - pkg/utils/validation/cidr
+            - pkg/utils/validation/features
+            - pkg/utils/version
+            - vendor
+            - VERSION
+        ldflags:
+          - '{{.LD_FLAGS}}'
+        main: ./cmd/gardener-resource-manager
 deploy:
   helm:
     releases:
-    - name: gardener-operator
-      chartPath: charts/gardener/operator
-      namespace: garden
-      createNamespace: true
-      wait: true
-      artifactOverrides:
-        image: eu.gcr.io/gardener-project/gardener/operator
-        # dummy artifact overrides to make skaffold load images into cluster
-        resource-manager:
-          image: eu.gcr.io/gardener-project/gardener/resource-manager
-      setValueTemplates:
-        # on a local cluster, skaffold uses a local immutable tag for artifacts,
-        # overwrite our image vector to use the image that was just built
-        # see https://skaffold.dev/docs/pipeline-stages/taggers/#how-tagging-works
-        imageVectorOverwrite: |
-          images:
-          - name: gardener-resource-manager
-            repository: localhost:5001/eu_gcr_io_gardener-project_gardener_resource-manager
-            tag: "{{.IMAGE_TAG2}}"
-        env:
-        - name: GARDENER_OPERATOR_LOCAL
-          value: "true"
-      imageStrategy:
-        helm: {}
+      - name: gardener-operator
+        chartPath: charts/gardener/operator
+        namespace: garden
+        setValueTemplates:
+          env[0].name: GARDENER_OPERATOR_LOCAL
+          env[0].value: "true"
+          image.repository: '{{.IMAGE_REPO_eu_gcr_io_gardener_project_gardener_operator}}'
+          image.tag: '{{.IMAGE_TAG_eu_gcr_io_gardener_project_gardener_operator}}@{{.IMAGE_DIGEST_eu_gcr_io_gardener_project_gardener_operator}}'
+          imageVectorOverwrite: |
+            images:
+            - name: gardener-resource-manager
+              repository: localhost:5001/eu_gcr_io_gardener-project_gardener_resource-manager
+              tag: '{{.IMAGE_TAG_eu_gcr_io_gardener_project_gardener_resource_manager}}@{{.IMAGE_DIGEST_eu_gcr_io_gardener_project_gardener_resource_manager}}'
+          resource-manager.image.repository: '{{.IMAGE_REPO_eu_gcr_io_gardener_project_gardener_resource_manager}}'
+          resource-manager.image.tag: '{{.IMAGE_TAG_eu_gcr_io_gardener_project_gardener_resource_manager}}@{{.IMAGE_DIGEST_eu_gcr_io_gardener_project_gardener_resource_manager}}'
```

Comment:
What is this needed for?

Reply:
It was added by `skaffold fix` and I did not clean it up :sweat_smile: